drizzlepac/haputils/catalog_utils.py

```diff
@@ -1656,6 +1652,11 @@ def identify_sources(self, **pars):
                 # Only pass along the array to be consistent with the g2d_kernel object
                 rw2d_kernel = rw2dk.array
 
+                # Write out diagnostic data
+                if self.diagnostic_mode:
+                    outname = self.imgname.replace(".fits", "_rwkernel.fits")
+                    fits.PrimaryHDU(data=rw2d_kernel).writeto(outname)
+
```

Comment:
I wondered whether the diagnostic_mode should also write out the Gaussian kernel to a fits file?  Maybe that already happens and I have not encountered it.  But it seems odd to write out only the RW kernel.

Reply:
@rlwastro 
The Gaussian kernel is written out to a file in diagnostic mode (has been for a long time):
hst*_kernel.fits vs hst*_rwkernel.fits.